docker-compose.yml

```diff
@@ -0,0 +1,41 @@
+version: '3'
+services:
+  nginx:
```

Comment:
If you will have more than 1 docker with the same names of services, you can have warnings about orphaned services. You can `COMPOSE_PROJECT_NAME=your_app_name` to your env variables used by docker.

Reply:
.